index.Rmd

```diff
@@ -57,7 +57,7 @@ map_1 <- tm_basemap(leaflet::providers$Esri.WorldTopoMap, alpha = 0.4) +
 map_1
```

Comment:
It would be really helpful as a reader if your subheadings like "show whole data set" or "Process and spatialize data" were bolded or italicized or entered in as subheadings to differentiate them visually from bigger blocks of text.